packages/xo-web/src/common/xo/index.js

```diff
@@ -62,6 +62,10 @@ export const XEN_VIDEORAM_VALUES = [1, 2, 4, 8, 16]
 
 // ===================================================================
 
+export const VM_BOOT_FIRMWARES = ['bios', 'uefi']
```

Comment:
Is there a documentation somewhere containing the possible values?

/cc @olivierlambert

Reply:
There is indeed an incorrect official doc here: https://docs.citrix.com/en-us/citrix-hypervisor/whats-new/experimental.html

The example given in `xe` is wrong, it's `uefi` and `bios` (not `UEFI` or `BIOS`)

Reply:
IMHO we should add a comment in the code (either server or web side) with the link and the errata.

Reply:
Please add a comment explaining the XAPI values should be lowercased.